lib/ecto/changeset.ex

```diff
@@ -2976,7 +2981,7 @@ defmodule Ecto.Changeset do
         end
       {field, {tag, %{cardinality: :one}}}, acc when tag in @relations ->
         if changeset = Map.get(changes, field) do
-          case traverse_errors(changeset, msg_func) do
+          case traverse_function.(changeset, msg_func) do
             errors when errors == %{} -> acc
             errors -> Map.put(acc, field, errors)
```

Comment:
```suggestion
            items when items == %{} -> acc
            items -> Map.put(acc, field, items)
```
Doesn't refer to errors only anymore, so guess these should be renamed. Likely theres a better option than `items` tho.